FlowCryptUI/Cell Nodes/RecipientEmailsCellNode.swift

```diff
@@ -55,15 +61,8 @@ final public class RecipientEmailsCellNode: CellNode {
         guard recipients.isNotEmpty else {
             return ASInsetLayoutSpec(insets: .zero, child: collectionNode)
         }
-        let recipientNodeInset = RecipientEmailNode.Constants.layoutInsets.height
-            + RecipientEmailNode.Constants.titleInsets.height
 
-        let textSize: CGSize = recipients.first?.email.size() ?? .zero
-        let recipientsHeight = (textSize.height + recipientNodeInset) * CGFloat(recipients.count)
-        let insets = Constants.minimumLineSpacing * CGFloat(recipients.count - 1)
-        let height = recipientsHeight + insets + Constants.sectionInset.height
-
-        collectionNode.style.preferredSize.height = height
+        collectionNode.style.preferredSize.height = collectionLayoutHeight
```

Comment:
@sosnovsky note for reviewing:
Calculation for flow layout height happens after we set height for collectionNode, so it is not possible to calculate height here( 

Thats why we need to observe calculated height from collection layout in controller to be able to react on height changes and update `RecipientEmailsCellNode` with updated height value

Reply:
If it's generally surprising for the person reading the code, it may be appropriate to add a comment in the code.

Reply:
Have just added comment's for reviewers, don't think it worth commenting it.